docs/getting-started.rst

```diff
@@ -25,11 +25,12 @@ Install
 
 The CrateDB Python client is `available`_ as a `PyPI`_ package.
 
-To install the package, run:
```

Comment:
We could maybe also mention both options?
I don't know what is more common.

Reply:
I would just leave it away to reduce noise and keep it minimal.